### ISSUE TITLE
`Invalid argument: idToken has no "kid" claim.` when using auth emulator

### ISSUE THREAD

**Issue by jwarby** [awinogrodzki/next-firebase-auth-edge#82]
Hi, apologies if the fix for this is noted somewhere, but I couldn't find any references to it.

I'm getting this error when trying to login using the auth emulator (`/api/login` is crashing - 500 internal server error):

```
- error ../node_modules/.pnpm/next-firebase-auth-edge@0.8.1_next@13.4.13/node_modules/next-firebase-auth-edge/lib/auth/token-verifier.js (60:18) @ FirebaseTokenVerifier.verifyContent
- error Invalid argument: idToken has no "kid" claim.
```

It's working OK when I point to a real Firebase instance though. Any ideas what this could be?

**Comment by jwarby** [comment]
Update: I had a typo in my env var name:

 ```diff
 - FIREBASE_AUTH_EMULATOR_URL=localhost:9099
 + FIREBASE_AUTH_EMULATOR_HOST=localhost:9099
 ```
 
 After I fixed that, I'm getting a _slightly_ different error:
 
 ```
 - error ../node_modules/.pnpm/next-firebase-auth-edge@0.8.1_next@13.4.13/node_modules/next-firebase-auth-edge/lib/auth/token-verifier.js (72:15) @ FirebaseTokenVerifier.mapJoseErrorToAuthError
- error Internal error: no-kid-in-header-error
```

**Comment by awinogrodzki** [comment]
Hey @jwarby!

Thanks for reporting. I fixed the error in `next-firebase-auth-edge@0.8.3`. 

Let me know if that works

**Comment by jwarby** [comment]
Hey @awinogrodzki - that's working now! :raised_hands: thank you so much!
